frontend/lib/auth/index.ts

```diff
@@ -51,7 +51,7 @@ const getOnBoardingCookie = (isModerator: boolean) => {
 const getToken = () => cookie.get(COOKIE_TOKEN);
 const getRefreshToken = () => cookie.get(COOKIE_REFRESH);
 
-const getAuthToken = async (force = false) => {
+const getAuthToken = async (force = false, roomName = '') => {
```

Comment:
I wouldn't set this to empty string by default. I'd let it undefined and typed with ` : string`
```suggestion
const getAuthToken = async (force = false, roomName: string) => {
```

Reply:
Take a look at how it is on the typescript handbook: https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters

Normally they define optional parameters as ?string, so we know it is string but it is optional to pass it.